gateway/ln-gateway/src/lightning/ldk.rs

```diff
@@ -414,6 +414,29 @@ impl ILnRpcClient for GatewayLdkClient {
             })
     }
 
+    async fn withdraw_onchain(
+        &self,
+        address: Address,
+        amount: BitcoinAmountOrAll,
+        _fee_rate_sats_per_vbyte: u64, // TODO: Respect this fee rate.
```

Comment:
Note for reviewers: `ldk-node` v0.3 doesn't support setting a fee-rate, but support for this is planned for v0.4 and is tracked here: https://github.com/lightningdevkit/ldk-node/issues/176

Reply:
Maybe link the issue directly in the code?

Reply:
What feerate will it use instead? Something from esplora?

Reply:
Looks like it uses esplora to attempt a 6-block conf: https://github.com/lightningdevkit/ldk-node/blob/7eca7fbc66c7430ac488ca80b69db2e1dea40f34/src/fee_estimator.rs#L103-L126

And falls back to 20 sat/vbyte: https://github.com/lightningdevkit/ldk-node/blob/7eca7fbc66c7430ac488ca80b69db2e1dea40f34/src/fee_estimator.rs#L164